plugin.video.catchuptvandmore/resources/lib/channels/be/rtlplaybe.py

```diff
@@ -209,9 +209,9 @@ def list_videos_search(plugin, search_query, item_id, page, **kwargs):
                                       item_id=item_id,
                                       program_id=search_id)
                     item_post_treatment(item)
+                    yield
```

Comment:
looks like a leftover, you yield the item in the line below

Reply:
Thanks you again